app/root.tsx

```diff
@@ -13,10 +13,7 @@ export default function App() {
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width,initial-scale=1" />
         <link rel="preconnect" href="https://cdn.shopify.com/" />
-        <link
-          rel="stylesheet"
-          href="https://cdn.shopify.com/static/fonts/inter/v4/styles.css"
-        />
+        <script src="https://cdn.shopify.com/shopifycloud/app-bridge-ui-experimental.js"></script>
```

Comment:
This is meant to be included in `app-bridge-ui-experimental.js` but there is currently an issue https://github.com/Shopify/admin-ui-foundations/issues/2741